README.md

```diff
@@ -1,3 +1,45 @@
 # NYPL Core
 
-Models, mappings, and vocabularies for the NYPL Core ontology.  
\ No newline at end of file
+Models, mappings, and vocabularies for the NYPL Core ontology.
+
+## Deployment Process
+
+### General Workflow for Changes
+
+* Create a new branch from master
+* Change the CSV file(s) with the intended changes
+* Serialize changes via the appropriate script(s)
+* Commit changes
+* Create a pre-release tag in the form vX.Xa, i.e. v1.1a
+
+#### For QA:
+
+* Test the pre-release tag on AWS nypl-sandbox by setting the
+  NYPL_CORE_VERSION variable to vX.Xa for applications that need to test
+  the changes
+* Change environment variables for discovery-api (Elastic Beanstalk)
+* Apply the changes which will trigger an application restart
+* If no changes to variables are detected, a restart will not take
+  place. One must force a restart to clear caches and pull in changes
+* Test the changes on our QA server
+* Make adjustments to the mapping as needed
+* Create a new tag vX.Xb
```

Comment:
I think it's good to call out that one can make additional pre-release tags if needed, but that seems like a unique scenario. If adjustments to one's pre-release version are required, wouldn't one normally just move the pre-release tag to point to the patched code? In other words, if I've created a pre-release tag `X.Xa` and then notice an issue, given that I'm the only one interested in `X.Xa`, wouldn't I just update `X.Xa`?

Reply:
I think it makes sense to increment the tag when there's more than one person involved (ex. vX.Xa fails testing, I need to reserialize, and Greg needs to re-test), but agree with Paul that if it's just one person involved, maybe not necessary.

Reply:
For adding new pre-release tags, I was thinking more about releases which should be fixed, unchanged versions of the code. Since these are pre-release tags, I don't see why they can't just be updated to keep changes together in the same pre-release tag. I'll add a line about using `git tag -f vX.Xa` when introducing patched code to the pre-release tag.